eclass/tree-sitter-grammar.eclass

```diff
@@ -206,6 +208,15 @@ tree-sitter-grammar_src_install() {
 
 		dolib.so "${WORKDIR}/${soname}"
 		dosym "${soname}" /usr/$(get_libdir)/lib${PN}$(get_libname)
+		keepdir /usr/$(get_libdir)/tree-sitter
+		dosym ../"${soname}" \
+			/usr/$(get_libdir)/tree-sitter/${PN##tree-sitter-}$(get_libname)
+
+		if [[ -d "${S}/../queries" ]]; then
+			keepdir /usr/share/tree-sitter
+			insinto /usr/share/tree-sitter
+			doins -r "${S}/../queries"
+		fi
```

Comment:
```suggestion
        # Install queries (e.g. highlight.scm) so that they can be found by NeoVim.
		if [[ -d "${S}/../queries" ]]; then
			keepdir /usr/share/tree-sitter
			insinto /usr/share/tree-sitter
			doins -r "${S}/../queries"
		fi
```